### PR TITLE
My Site Dashboard (Phase 2): Fix segmented control alignment in landscape mode

### PR DESCRIPTION
**Pull request by momo-ozawa**
## Description
- Fixes constraints for the segmented control so that it displays with the correct alignment in landscape mode
- Handles dynamic type for segmented control

Before | After
-- | --
Simulator Screen Shot - iPhone 13 - 2022-02-16 at 16 10 54  | Simulator Screen Shot - iPhone 13 - 2022-02-16 at 16 07 15

Dynamic Type (Normal) | Dynamic Type (Large)
-- | --
 | 

<details>
 <summary>Screen recording below</summary>

https://user-images.githubusercontent.com/6711616/154309128-2f5c0800-d655-4cc8-8bf8-4654d3d1633f.mp4

</details>

## How to test

**Make sure to run your sandbox with the 1.1 API diff applied and to have the My Site Dashboard flag enabled:**

1. Open the app
2. Go to My Site
3. Change your device orientation to landscape
4. ✅ The segmented control should be aligned with the rest of the elements displayed on My Site (i.e. Site icon, Quick Actions cell)
5. Change the system font size (via the Environment Overrrides menu in Xcode, for example)
6. ✅ The font size for the segmented control should change

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by momo-ozawa** [comment]
FYI @pachlava -- pinging you for visibility

**Comment by pachlava** [comment]
Did a test on iPhone 13 Simulator - works as advertised. Thanks for pinging me @momo-ozawa 👋 

One side question that came to my mind while I tested: should (or technically can) the text in segmented control follow the dynamic text sizing?

**Comment by momo-ozawa** [comment]
@pachlava 

> One side question that came to my mind while I tested: should (or technically can) the text in segmented control follow the dynamic text sizing?

Now it is! 299f6c9

File: WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift [hunk 1]
```diff
@@ -149,6 +150,13 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         setupTransparentNavBar()
     }
 
+    private func subscribeToContentSizeCategory() {
+        NotificationCenter.default.addObserver(self,
```
Comment (leandroalonso) [inline WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift:158]: As a good practice, what do you think about unsubscribing to notifications on `deinit`?

**Review by leandroalonso — APPROVED** [review]
Left a small comment about `deinit`, besides that all goodio!

File: WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift (hunk 1 above)
Comment (momo-ozawa) [reply WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift:158]: Fixed! 49fa691

**Pull request merged**